apps/shell/tash_main.c

```diff
@@ -220,10 +220,13 @@ static int tash_main(int argc, char *argv[])
 int tash_start(void)
 {
 	int pid;
+	int errcode;
 
 	pid = task_create("tash", TASH_TASK_PRIORITY, TASH_TASK_STACKSIZE, tash_main, (FAR char *const *)NULL);
 	if (pid < 0) {
-		printf("TASH is not started, error code = %d\n", pid);
+		errcode = errno;
+		DEBUGASSERT(errcode > 0);
```

Comment:
What's the purpose of this line?

Reply:
It validates that whether error code is junk value or not.  Because we return negative value of error code, its expected to be always greater than 0

Reply:
Is it possible to be negative value?

Reply:
@sunghan-chang , logically by software its not possible. only in case of memory corruption or hardware errors like BITFLIP could make it

Reply:
Do we need to concern hw error especially here?

Reply:
I could see that, It's been handled at other place as well . 
For ex. work_usrstart () in lib/libc/wqueue/work_usrthread.c  , Anyway if you think that it's not necessary, we can remove it

Reply:
Thank you for letting me know the places. I agree and merged.